src/Domain/OrientationMapHelpers.cpp

```diff
@@ -324,3 +324,38 @@ template void orient_each_component(
     const gsl::span<const std::complex<double>>& variables,
     const size_t num_pts, const std::vector<size_t>& oriented_offset) noexcept;
 }  // namespace OrientationMapHelpers_detail
+
+template <size_t VolumeDim>
+std::vector<double> orient_variables(
```

Comment:
For my understanding, can you explain the use-case for the `std::vector`? I would have expected that most of the time this sort of utility would be applied to a `Variables`.

Reply:
I have added a description of the use case into the documentation (DG-subcell I don't know at compile-time what will be sent with certainty). Please let me know if that's a compelling case and is described clearly enough.